test/about/historyTest.js

```diff
@@ -132,14 +136,12 @@ describe('about:history', function () {
     })
     it('selects multiple contiguous rows when shift clicked', function * () {
       yield this.app.client
-        .tabByUrl(aboutHistoryUrl)
-        .loadUrl(aboutHistoryUrl)
         .click('table.sortableTable td.title[data-sort="Brave"]')
         .keys(Brave.keys.SHIFT)
         .click('table.sortableTable td.title[data-sort="https://www.youtube.com"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="Brave"]')
         .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://www.youtube.com"]')
-        .waitForVisible('table.sortableTable tr.selected td.title[data-sort="https://brave.com/test"]')
+        .waitForVisible('table.sortableTable td.title[data-sort="https://brave.com/test"]')
```

Comment:
By the way I see that in TraviCI tests for about:history are passed.

Tried locally, but get errors:

1) about:history page content "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19

  2) about:history double click behavior "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19

  3) about:history multi-select behavior "before all" hook:
     WaitUntilTimeoutError: Promise was rejected with the following reason: Error: unknown error: Cannot read property 'appStoreRenderer' of undefined
      at execute(<Function>) - brave.js:507:19